datadog_checks_base/datadog_checks/base/checks/openmetrics/mixins.py

```diff
@@ -630,6 +635,29 @@ def _join_labels(self, metric, scraper_config):
                     if mapping_key in label_mapping and mapping_value in label_mapping[mapping_key]:
                         sample_labels.update(label_mapping[mapping_key][mapping_value])
 
+    def _ignore_metrics_by_label(self, scraper_config, metric_name, sample):
+        ignore_metrics_by_label = scraper_config['ignore_metrics_by_labels']
+        sample_labels = sample[self.SAMPLE_LABELS]
+        for label_key, label_values in ignore_metrics_by_label.items():
+            if not label_values:
+                self.log.warning(
```

Comment:
curious if this might be a bit noisy? maybe debug is better?

Reply:
that sounds good, updated